tree/treeplayer/src/TTreeProcessorMT.cxx

```diff
@@ -525,43 +525,35 @@ void TTreeProcessorMT::Process(std::function<void(TTreeReader &)> func)
                                                           fFileNames, clusterAndEntries.second);
    }
 
-   const auto &clusters = clusterAndEntries.first;
-   const auto &entries = clusterAndEntries.second;
```

Comment:
I think these are still useful to make the following code more readable.

Probably `clusterAndEntries` should be called `allClustersAndEntries` and the variables should be called `allClusters` and `allEntries`.